tribits/core/package_arch/TribitsAddTest.cmake

```diff
@@ -336,7 +336,7 @@ include(TribitsAddTestHelpers)
 #     send to stdout.  Otherwise, the test will fail.  This is set using the
 #     built-in CTest property ``PASS_REGULAR_EXPRESSION``.  Consult standard
 #     CMake documentation for full behavior.  TIPS: Replace ';' with '[;]' or
-#     CMake will interpret this as a array element boundary.  To match '.',
+#     CMake will interpret this as an array element boundary.  To match '.',
```

Comment:
Wow, codespell can take into the account the context of the sentence as well to suggest updated works.  It is not just looking at works in isolation.  Very nice!